inc/languages/english/member.lang.php

```diff
@@ -191,6 +191,8 @@
 $l['already_logged_in'] = "Notice: You are already currently logged in as {1}.";
 $l['admin_edit_in_acp'] = "Edit this user in Admin CP";
 $l['admin_ban_in_acp'] = "Ban this user in Admin CP";
+$l['admin_manageban_in_acp'] = "Lift ban in Admin CP";
+$l['admin_editban_in_acp'] = "Edit ban in Admin CP";
 $l['admin_options'] = "Administrator Options";
```

Comment:
Following existing pattern these should be:
`$l['lift_ban_in_acp']` & $l['edit_ban_in_acp']